### PR TITLE
add mi hello world sample

### PR DESCRIPTION
**Pull request by chameerar**

### CODE REVIEW

File: hello-world-mi/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+displayName: Hello World in MI
+description: Hellow World project created using WSO2 Micro Integrator
```
Comment (a5anka) [inline hello-world-mi/metadata.yaml:2]:
```suggestion
description: A Hello World project created using the WSO2 Micro Integrator
```

File: hello-world-mi/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+componentType: service
+buildPreset: mi
```
Comment (a5anka) [inline hello-world-mi/metadata.yaml:4]:
```suggestion
buildPreset: wso2-mi
```

Comment (chameerar) [reply hello-world-mi/metadata.yaml]: Fixed in dbebb9a56cd9f001df2b9b4cbb98397a9ada986f

File: hello-world-mi/metadata.yaml (hunk 1 above)
Comment (chameerar) [reply hello-world-mi/metadata.yaml]: Fixed in dbebb9a56cd9f001df2b9b4cbb98397a9ada986f

File: hello-world-mi/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+displayName: Hello World in MI
+description: A Hello World project created using the WSO2 Micro Integrator
```
Comment (a5anka) [inline hello-world-mi/metadata.yaml:2]:
```suggestion
description: A Hello World project created using the WSO2 Integration Studio
```

**Review by a5anka — APPROVED** [review]

**Pull request merged**